### PR TITLE
Zib-AbilityToDrink

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## AbilityToDrink (zib-AbilityToDrink, nl-core-AbilityToDrink)
+* No significant changes
```
Comment (ArdonToonstra) [inline release-notes.md:2]: Missing note for DrinkingLimitations profile

File: zib/StructureDefinitions/zib-AbilityToDrink.DrinkingLimitations.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="url"/>
+      <value value="https://www.nictiz.nl"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="DrinkingConstraints specifies the patient’s constraints in drinking."/>
```
Comment (ArdonToonstra) [inline zib/StructureDefinitions/zib-AbilityToDrink.DrinkingLimitations.xml:19]: The zib concept decription is : "DrinkingLimitations specifies the patient’s limitations in drinking."

**Review by ArdonToonstra — CHANGES_REQUESTED** [review]

File: release-notes.md (hunk 1 above)
Comment (Ashwin-nictiz) [reply release-notes.md:6]: Fixed!

File: zib/StructureDefinitions/zib-AbilityToDrink.DrinkingLimitations.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply zib/StructureDefinitions/zib-AbilityToDrink.DrinkingLimitations.xml]: Fixed!

**Review by ArdonToonstra — COMMENTED** [review]
Looks good now!

**Review by LuudSlagter — APPROVED** [review]

**Pull request merged**